src/Test.UnitTests.Sarif/Core/SarifLogTests.cs

```diff
@@ -170,6 +172,19 @@ public void SarifLog_SplitPerTarget_WithEmptyLocations()
             }
         }
 
+        [Fact]
+        public void SarifLog_LoadDeferred()
+        {
+            var random = new Random();
```

Comment:
>Random [](http://example.com/codeflow?start=29&length=6)

Don't ever introduce randomness into a method without capturing the seed. Otherwise we can't reproduce test failures.

Here's what to do:

Allocate a 4 byte array
Using RandomNumberGenerator.Fill to fill it
Convert this byte array to a number (using the Convert class)
Persist this int32 value to test output
Initialize your Random instance with that seed

Reply:
done